src/components/picker/index.tsx

```diff
@@ -173,14 +175,7 @@ const Picker = React.forwardRef((props: PickerProps, ref) => {
     onDoneSelecting
   ]);
 
-  const modalProps: ExpandableOverlayProps['modalProps'] = {
-    animationType: 'slide',
-    transparent: Constants.isIOS && enableModalBlur,
-    enableModalBlur: Constants.isIOS && enableModalBlur,
-    onRequestClose: topBarProps?.onCancel,
-    onShow,
-    ...pickerModalProps
-  };
+  const modalProps: ExpandableOverlayProps['modalProps'] = componentProps.pickerModalProps;
```

Comment:
Why here you assign component.xxx to a new const? It's hard to keep track where this is from

Reply:
Also you have a const under the same name (different scope sure) on line 195

Reply:
removed, passing `componentProps.pickerModalProps` directly to the `ExpandableOverlay` as `modalProps`